### ISSUE TITLE
dragon breeding issues

### ISSUE THREAD

**Issue by mser7** [Alex-the-666/Ice_and_Fire#552]
dragon priority fighting mobs over breeding
flying dragons refuse to breed
radius for dragons to notice their mate is too small, dragons can see must further than normal mobs.
dragons can't properly mate due to their gigantic hitbox when coming close together to breed.

**Comment by mser7** [comment]
forget, you can breed them again immediately after they ust mated.

**Comment by Jijuihin** [comment]
If I should have made a new issue for this, just let me know and I'll fill one out (and sorry for posting in the wrong place if so). 
However, I would just like to add to this that Ice Dragons seem to have a much harder time breeding than Fire Dragons in 1.4.0. 

After encountering issues with breeding in the server I play with my friends, I tested breeding in a world with just Ice and Fire/JEI/Hwyla as well, and out of 4 breeding pairs for stage 3, 4 and 5 dragons, only the stage 3 dragons managed to breed successfully. Of these, only 2 pairs were able to make an egg, and only 1 of the pairs was able to make an egg after this (despite several dozen attempts).
Having tested the same for Fire Dragons, where the dragons never failed to make an egg unless I tried breeding them straight after they had already made an egg, it seems the issue is with Ice Dragons specifically, not Dragons in general.
It could be that I'm not fulfilling some criteria, but I have tried both in and out of cold biomes, made sure to always have the dragon's hunger topped up and have tried breeding in stay mode and wandering mode, nothing has made a difference.
